native/packages/react-native-bpk-component-button/stories.js

```diff
@@ -215,6 +222,14 @@ storiesOf('BpkButton', module)
       {generateButtonStoryForType('featured')}
     </View>
   ))
+  .add('docs:withTheme', () => (
+    <BpkThemeProvider theme={themeAttributes}>
+      <ScrollView>
```

Comment:
This puts the themed buttons at the top of the page whereas other buttons pages are centred vertically.

Reply:
Thanks, now fixed.